### DISCUSSION TITLE
Some signature errors doing dmesg

### DISCUSSION THREAD

**Discussion by crazyrobinhood** [berarma/new-lg4ff#89] [Q&A] [answered]
[    4.741816] hid_logitech_new: loading out-of-tree module taints kernel.
[    4.741828] hid_logitech_new: module verification failed: signature and/or required key missing - tainting kernel
[    4.742586] logitech 0003:046D:C294.0001: fixing up Logitech Driving Force report descriptor

is this normal?
I cannot make steam recognize my wheel at all. (oversteer does)
any help appreciated!

**Comment by berarma** [comment, marked as answer]
It's a security measure that doesn't alter functionality. The message is only a warning.

I haven't ever worried about Steam recognizing my wheel. I just start the game and use it. I've just tested and mine isn't recognized either. I guess it's because it's not seen as a gamepad. It shouldn't prevent you from using it.

**Reply by crazyrobinhood** [reply]
Thank you for your reply!
The problem is with Assetto corsa unfortunately. using content manager the wheel doesn't show up in game.
My os is PopOS! with 6.6.6-76060606-generic kernel.
and I had to chmod 666 uinput (which is not safe) beacuse otherwise oversteer says I don't have permissions to modify. Like if udevs rules are not working correctly. Am I doing something wrong?

**Reply by berarma** [reply]
Latest release of Oversteer has an issue. You need to plug the wheel after booting on Linux so that the device permissons are correctly set. I'll do a new release shortly to fix it.

**Reply by crazyrobinhood** [reply]
Ah thank you for the info!

I know this is not the correct place to discuss singular games but I recently installed Assetto corsa with proton 8.27 GE using [this ](https://steamcommunity.com/app/244210/discussions/0/3824163953451160286/) guide.
I installed the content manager as well but no wheel for me. Honestly AC is the only game I'd like to play in linux with my wheel (an old g25). Mai I talk with someone somewhere :) Thanks in advance.

**Reply by berarma** [reply]
I suggest you look in the Proton repository. It has an issue open for each game. The ProtonDB might also have some info.

**Reply by crazyrobinhood** [reply]
Thank you for your answers, your work and your time!

**Reply by berarma** [reply]
New release 0.8.1 fixing the permissions.
